study-datastore/src/main/java/com/hphc/mystudies/dao/StudyMetaDataDao.java

```diff
@@ -740,11 +740,6 @@ public EligibilityConsentResponse eligibilityConsentMetadata(String studyId) thr
                           .replaceAll("em>", "i>")
                           .replaceAll("<a", "<a style='text-decoration:underline;color:blue;'"));
             }
-
```

Comment:
Please update file copyright to add Google LLC & `2020-2021`

Reply:
Updated copyright.